integration_tests/macros/my_metrics.sql

```diff
@@ -14,12 +14,25 @@
     )
 {% endmacro %}
 
-{% macro re_data_metric_my_custom_table_metric(time_column) %}
+{% macro re_data_metric_my_custom_table_metric(time_column, config) %}
     1000
 {% endmacro %}
 
 
-{% macro re_data_metric_distinct_count(column_name) %}
+{% macro re_data_metric_distinct_count(column_name, config) %}
     count(distinct( {{column_name}} ))
 
-{% endmacro %}
\ No newline at end of file
+{% endmacro %}
+
+{% macro re_data_metric_regexp_test(column_name, config) %}
```

Comment:
I believe in case of not passing variables, best would be to throw informative exception. I would go with this: https://docs.getdbt.com/reference/dbt-jinja-functions/exceptions

I see only compiler_error exception mentioned here, but not 100% it should be possibility to throw just standard exception.

Reply:
I agree, the only other option we seem to have is to log a warning.